src/pybind/mgr/dashboard/ci/cephadm/bootstrap-cluster.sh

```diff
@@ -11,7 +11,7 @@ mkdir -p /etc/ceph
 mon_ip=$(ifconfig eth0  | grep 'inet ' | awk '{ print $2}')
 
 bootstrap_extra_options='--allow-fqdn-hostname --dashboard-password-noupdate'
-bootstrap_extra_options_not_expanded='--skip-monitoring-stack'
+bootstrap_extra_options_not_expanded=''
```

Comment:
I disabled it previously because of the mgr restart that occurs when prometheus is started which interrupts the dashboard e2e. Now I disabled it found a workaround for it.